packages/components/src/components/Warning/Warning.tsx

```diff
@@ -29,6 +29,7 @@ export interface WarningProps {
     icon?: IconType;
     filled?: boolean;
     margin?: FrameProps['margin'];
+    'data-test'?: string;
```

Comment:
this really works? passing `data-tests` translates to `dataTest` on line 133? is it some react magic I am totally unaware of?